src/test-api/api.yaml

```diff
@@ -193,6 +195,30 @@ components:
         $ref: '#/components/schemas/Extension'
       title: ExtensionList
       description: Data model for the complex type ExtensionList
+    Account:
+      type: object
+      required:
+        - address
+        - currency
+        - description
+      properties:
+        address:
+          type: string
+          description: The routable address of this account.
+        currency:
+          type: string
```

Comment:
I think this should be the `CurrencyCode` enum, but I'm happy to leave it as is for now, since that enum would double the size of this file.

Reply:
Ok.